docs/csharp/language-reference/keywords/accessibility-levels.md

```diff
@@ -38,7 +38,11 @@ Use the access modifiers, `public`, `protected`, `internal`, or `private`, to sp
 |`struct`|`private`|`public`<br /><br /> `internal`<br /><br /> `private`|  
 
 \* An `interface` member with `private` accessibility must have a default implementation.
-
+  
+> [!NOTE]  
+> If a class or struct is modified with the [`record`](../builtin-types/record.md) keyword modifier, then the same access modifiers are allowed.  
+> However, with the [`record`](../builtin-types/record.md) modifier the default member accessibility is `internal` both for class and the struct.  
```

Comment:
This statement isn't correct. The default for members of a `record` is `private`, just like for members of a `class` or `struct`. 

What did you mean here? Can you give me an example that led to this?

Otherwise, let's remove the second sentence:

```suggestion
```

Reply:
Well, I must admit it's just a mistake of mine, I'm sorry 🙏
I just rechecked, you are right about the private for members by default as well.
As for the last sentence - shouldn't we change it to mention that the same (`private`) goes for members, instead of removing it? The first sentence doesn't mention the members default accessibility, only record itself.

Reply:
@BartoszKlonowski Yes, adding the correct default would be even better! Once you make the update, I'll :shipit:

Reply:
@BillWagner Sure, it's now corrected.